.rat-excludes

```diff
@@ -45,6 +45,9 @@ clients/*
 files/*
 dags/*
 airflow.iml
+airflow-core.iml
+devel-common.iml
+task-sdk.iml
```

Comment:
What are `.iml` files? Should this just be `*.iml`?

Reply:
Should work yeah. I think there is no other known `iml` extension.